atomic_reactor/tasks/sources.py

```diff
@@ -28,21 +28,13 @@ def source(self) -> source.Source:
         return source.DummySource(None, None, workdir=self.build_dir)
 
 
-class SourceBuildBaseTask(plugin_based.PluginBasedTask[SourceBuildTaskParams]):
-    """Base task for defining different build phases for source container build."""
-
-    def prepare_workflow(self) -> inner.DockerBuildWorkflow:
-        """After preparing the workflow as usual, fully initialize the root build dir.
-
-        Unlike the binary container container workflow, the platforms to be used for the build
-        are known in advance here (more accurately, source containers do not have a platform).
-        """
-        workflow = super().prepare_workflow()
-        workflow.build_dir.init_build_dirs(["noarch"], self._params.source)
-        return workflow
```

Comment:
Why was this removed?

Reply:
it's back

Reply:
Could you also put it back in the same place, so that the diff will show what changed?